src/main/scala/clump/ClumpContext.scala

```diff
@@ -6,7 +6,7 @@ import scala.collection.mutable.HashMap
 
 private[clump] final class ClumpContext {
 
-  private val fetchers =
+  private[this] val fetchers =
```

Comment:
It can't be accessed from other instances of the same type.

```scala
class A { 
    private[this] val a = "a" 

   def test(otherInstance: A) = otherInstance.a // doesn't compile!
}

```